package.json

```diff
@@ -30,6 +30,7 @@
     "cli-table3": "^0.5.1",
     "debug": "^4.1.0",
     "ember-cli-babel": "^7.8.0",
+    "ember-cli-htmlbars": "^3.0.1",
```

Comment:
Was it necessary to add this to dependencies?

Reply:
no, moved it back to devDependency :)